### ISSUE TITLE
Compact display mode

### ISSUE THREAD

**Issue by Askich239** [KRTirtho/spotube#1646]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When toggling to the compact display mode, impossible to locate the Settings button

### Expected Behavior

Being able to try the different display mode to choose the adequate one

### Steps to reproduce

1. Clic on Settings
2. Change display mode to Compact
3. When returning to home screen, you won't find the settings button nor the profil

### Operating System

Ubuntu 24.04 LTS

### Spotube version

3.7.1

### Installation source

Flathub (Flatpak)

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by SMGolamAlif** [comment]
Same problem here!  I just changed my Change display mode to Compact and am now stuck. I am unable to go to the sitting page to fix it. There should be another tab for sittings in compact mode. 
Hey, [Askich239] have you fixed it anyhow? or do I need to reinstall the whole application?

**Comment by AimingDuck** [comment]
Stumbled upon the same issue, 'ctrl + w' is keybind to open settings, at least on linux

**Comment by peterpandemonium** [comment]
> Stumbled upon the same issue, 'ctrl + w' is keybind to open settings, at least on linux

Appreciate for sharing this. Oddly, it's 'Ctrl + ,' for me on EndeavorOS (Arch).

**Comment by AimingDuck** [comment]
> > Stumbled upon the same issue, 'ctrl + w' is keybind to open settings, at least on linux
> 
> Appreciate for sharing this. Oddly, it's 'Ctrl + ,' for me on EndeavourOS (Arch).

That's my fault lol, completely forgot I was on a non-qwerty keyboard layout. Thanks for posting the actual keybind :)

**Comment by LinearAmendment** [comment]
Hey just found the keybind for windows it is ' Ctrl + , '. Hope it helps :)

**Comment by SMGolamAlif** [comment]
I am still stuck in Compact mode. I am using KDE Neon 6.0 (Debian-based).  Both "Ctrl  + ," and  "Ctrl + w ". didn't work for me .

**Comment by ishrafislam** [comment]
Yes, when switched to compact mode settings button is not shown anywhere. Best UX will be to add another tab for settings. Although "Ctrl + ," works there should be a visible button.

- OS: Windows 11
- Version: 3.7.0
